hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/forensics/OrderedComparison.java

```diff
@@ -73,9 +73,28 @@ public static List<DifferingEntries> findDifferencesBetweenV6(
         System.out.println("Parsing stream @ " + secondStreamDir);
         final var secondEntries = parseV6RecordStreamEntriesIn(secondStreamDir, watchingPredicate);
         System.out.println(" ➡️  Read " + secondEntries.size() + " entries");
+        alignConsensusTimes(firstEntries, secondEntries);
         return diff(firstEntries, secondEntries, recordDiffSummarizer);
     }
 
+    private static void alignConsensusTimes(
+            final List<RecordStreamEntry> firstEntries, final List<RecordStreamEntry> secondEntries) {
+        int i = 0;
+        int j = 0;
+        // We assume that only second entries list will be missing entries
```

Comment:
Curious, why can we make this assumption?

Reply:
This isonly based on ethe evidence we saw until now in differential testing for 1 minute. There were always missed entries in mod-service generated records